docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:7
```

Comment:
@newPOPE great, feel free to improve anything after this is merged. I also agree the entrypoint is too complex